demo/src/main/res/values/controls_styles.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+  <style name="Controls" />
+
+  <style name="Controls.ScrubBar">
+    <item name="android:focusable">true</item>
+    <item name="android:indeterminateDrawable">@drawable/progress</item>
+    <item name="android:indeterminateOnly">false</item>
+    <item name="android:maxHeight">2dp</item>
```

Comment:
Why not just setting a fixed `layout_height` rather than setting equal min and max height?

Reply:
ah, true. 👍

Reply:
I can't do this. I'm using weights for the width so if I use `2dp` then it swallows the drawable for the thumb 🤔

Reply:
But this is the height... I'm not following I'm afraid, but I trust you 👍